.github/workflows/pipeline-segment-electron-test.yml

```diff
@@ -34,6 +39,7 @@ env:
   ELECTRON_OUT_DIR: Default
   ELECTRON_RBE_JWT: ${{ secrets.ELECTRON_RBE_JWT }}
   ELECTRON_GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+  IS_ASAN: ${{ inputs.is-asan }}
```

Comment:
does this have to be an env var? or can we use `${{ inputs.is-asan }}` directly instead? Env vars can make things hard to track where they're coming from, we should probably prefer to use inputs explicitly

Reply:
Looks like we use this in a couple of tests:
- https://github.com/electron/electron/blob/040acaaf3026aec8148aa010283e650e629d682b/spec/asar-spec.ts#L117
- https://github.com/electron/electron/blob/a0a13ad623d2f0beae8b2574a22cf012100e6b6b/spec/api-app-spec.ts#L1488
- https://github.com/electron/electron/blob/67894f1493ca5bdbd5c3f22d6ea8f5db7f283950/spec/spellchecker-spec.ts#L16

We can probably get rid of it eventually but we probably do need it for now 🤔

Reply:
Ah gotcha, can we use the input at least in the rest of the yml file?